test/check-machines-storage-pools

```diff
@@ -477,7 +477,7 @@ class TestMachinesStoragePools(machineslib.VirtualMachinesCase):
                 b.wait_not_present("#create-storage-pool-dialog")
 
         # Prepare a Volume Group to be used as source for the LVM pool
-        m.execute(f"wipefs -a {dev}; pvcreate {dev}; vgcreate vol_grp1 {dev}")
+        m.execute(f"wipefs -a {dev}; sleep .5; pvcreate {dev}; vgcreate vol_grp1 {dev}")
```

Comment:
Meh, this is begging for test flakes; how does it fail? could you rather do something like `until pvcreate ...; do sleep 0.5; done`?

Reply:
Yep that's much better, I'll get it amended